src/librustc/mir/mod.rs

```diff
@@ -116,6 +118,9 @@ pub struct Body<'tcx> {
     /// The layout of a generator. Produced by the state transformation.
     pub generator_layout: Option<GeneratorLayout<'tcx>>,
 
+    /// The generator kind.
```

Comment:
Please add a note here explaining what the purpose of the field is.

Reply:
Will do